templates/bundles/PrestaShopBundle/Admin/TwigTemplateForm/entity_search_input.html.twig

```diff
@@ -48,7 +50,9 @@
       {%- set id = form.vars.id ~ '_search_input' -%}
       <input {{ block('widget_container_attributes') }} />
     </div>
+    {% endif %}
```

Comment:
We don't have a twig linter unfortunatly but I hate when conditions arent aligned as HTML tags, it's way less readable to me...

Reply:
We already had this discussion and we don't see things the same way ^^ But I'm not gonna argue with you for this, especially since it seems to cause you physical pain 😅